soc/alif/ensemble/Kconfig.defconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2025 Alif Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_FAMILY_ENSEMBLE
```

Comment:
You are already inside the ensemble-directory so checking for ensemble looks a bit weird. I would say that you should have one more config in _soc/alif/_ where you pick the device family specific Kconfig - this file or a similar one for Balletto.

Reply:
It was pointed out to me that other vendors are doing this the same way.